pkg/scheduler/framework/preemption/preemption_test.go

```diff
@@ -284,7 +284,8 @@ func TestDryRunPreemption(t *testing.T) {
 			ctx, cancel := context.WithCancel(context.Background())
```

Comment:
```suggestion
			ctx, cancel := context.WithCancel(ctx)
```

The one coming from `NewTestContext`

Reply:
Updated.